### ISSUE TITLE
Unknown client HTTP error after logging in via Auth0

### ISSUE THREAD

**Issue by dannief** [NakedObjectsGroup/NakedObjectsFramework#142]
I have followed the steps outlined in the manual to enable Auth0. When I launch the application the rest service (http://localhost:5000) launches with a 401 error. After navigation to the application URL (http://localhost:5000), I am presented with the login screen where I can successfully authenticate via Google and Facebook. However the app redirects and displays a page with text below.
```
Error message received from server

Message: Unknown client HTTP error

Code: Unauthorized(401)

Description: An HTTP error code has been received from the server You can look up the meaning of this code in the Restful Objects specification..

Stack Trace :
No stack trace available
```

I've determined the error occurs on the following line (33) in the `JsonWebToken.cs` file
```c#
var payloadJson = JWT.JsonWebToken.Decode(token, secret, verify: true);
```
and the following exception is thrown:
```
JWT.SignatureVerificationException: Algorithm not supported.
   at JWT.JsonWebToken.GetHashAlgorithm(String algorithm)
   at JWT.JsonWebToken.Decode(String token, Byte[] key, Boolean verify)
   at NakedObjects.Template.App_Start.JsonWebToken.ValidateToken(String token, String secretKey, String audience, Boolean checkExpiration, String issuer, Boolean isSecretBase64Encoded) in C:\Dev\Personal\Projects\NofExpenseReport\Template.Server\App_Start\JsonWebToken.cs:line 33
   at NakedObjects.Template.App_Start.JsonWebTokenValidationHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken) in C:\Dev\Personal\Projects\NofExpenseReport\Template.Server\App_Start\JsonWebTokenValidationHandler.cs:line 59
```

**Comment by richardpawson** [comment]
See my comment on #141

**Comment by richardpawson** [comment]
Fixed in 9.0.5, now released.
